### PR TITLE
Rework config/html. Also remove cheerio

### PR DESCRIPTION
**Pull request by andy128k**
Related to #246

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -58,9 +58,7 @@ function favicons(source, options = {}, next) {
   }
 
   function createHTML(platform) {
-    return Promise.all(
-      Object.values(config.html[platform] || {}).map(µ.HTML.parse)
-    );
+    return Promise.all((config.html[platform] || []).map(µ.HTML.substitute));
```
Comment (brunocodutra) [inline src/index.js]: Minor, but I'm not sure `substitute` is a good name, maybe `render`?

Comment (brunocodutra) [inline src/index.js]: Minor, but I'm not sure `substitute` is a good name, maybe `render`?

File: src/helpers.js [hunk 2]
```diff
@@ -101,40 +100,8 @@ module.exports = function(options) {
...
-            isLink &&
-            tag.attr("rel") === "manifest" &&
-            options.loadManifestWithCredentials
-          ) {
-            tag.attr("crossOrigin", "use-credentials");
-          }
-
-          return resolve($.html(tag));
-        });
+      substitute(htmlFunction) {
```
Comment (brunocodutra) [inline src/helpers.js]: In the context of renaming `substitute` to `render` (see other comment) I'd rename `htmlFunction` to `htmlTemplate`, what do you think?

File: src/index.js (hunk 1 above)
Comment (andy128k) [reply src/index.js]: Render is much better!

File: src/helpers.js (hunk 2 above)
Comment (andy128k) [reply src/helpers.js]: Done

**Pull request merged**